contracts/interfaces/ICover.sol

```diff
@@ -9,27 +9,44 @@ interface ICover is IERC721 {
   /* ========== DATA STRUCTURES ========== */
 
   struct StakingPool {
```

Comment:
suggested rename in offline discussions: `StakingPoolAllocation` / something similar

Reply:
I vote for `CoverChunk`!